graph_altimeter/scan/neptune_client.py

```diff
@@ -18,18 +18,16 @@
 logger.addHandler(logging.NullHandler())
 
 
+# pylint: disable=arguments-differ
 class GraphAltimeterNeptuneClient(AltimeterNeptuneClient):
-    """Client to run store the a graph generated by Altimeter into a Neptune
+    """Client to store a graph generated by Altimeter into a Neptune
     compatible database following the Security Graph Universe schema.
 
     Args:
-        max_age_min: maximum acceptable age in minutes of graphs.  Only graphs
-        which are found
-                     that meet this critera will be queried.
-        neptune_endpoint: NeptuneEndpoint object for this client
-    """
+        neptune_endpoint: NeptuneEndpoint object for this client"""
 
     def __init__(self, endpoint: NeptuneEndpoint):
+        # We the max_age_min is not used lpg's graphs.
```

Comment:
Do you mean "The max_age_min parameter is not used by lpg graphs."?

Reply:
yep, bad wording